gitter/score.py

```diff
@@ -0,0 +1,62 @@
+from gitter.splitter import Splitter
+from gitter.pos_tagger import POSTagger
+from gitter.dict_tagger import DictionaryTagger
+
+
+def tagged_sentences(text):
+    """
+    Tag a message with dict_tagged_sentence.
+
+    This method takes a string parameter which is the message
+    send by the newcomer and then it split it into list of lists
+    of words using the Splitter class, and it uses POSTagger class
+    to tag each word with pos. And then it uses the DictionaryTagger
+    class and the two dict 'gitter/question.yml' and 'gitter/ignore.yml'
+    and tag the pos tagged word with 'question' or 'ignore'.
+
+    :param text: a string representing the message.
+                 e.g. 'How to solve this issue?'
+    :return: a list of words tagged with DictTagger and POSTagger.
+             e.g. [[('how', 'how', ['question', 'WRB']),
+                    ('can', 'can', ['MD']),
+                    ('I', 'I', ['PRP']),
+                    ('solve', 'solve', ['VB']),
+                    ('this', 'this', ['DT']),
+                    ('issue', 'issue', ['NN']),
+                    ('?', '?', ['question', '.'])]]
+    """
+    splitter = Splitter()
+    postagger = POSTagger()
+    splitted_sentences = splitter.split(text)
+    pos_tagged_sentences = postagger.pos_tag(splitted_sentences)
+    dicttagger = DictionaryTagger(['gitter/question.yml', 'gitter/ignore.yml'])
```

Comment:
We need one Splitter, one POSTagger and one DictionaryTagger.
Do not recreate these objects every time you want to do some nlp on a fragment.

Create another class which creates and uses those objects internally, and instantiate only one instance of our class.